tests/rados/test_scrub_log.py

```diff
@@ -17,94 +18,221 @@
 
 def run(ceph_cluster, **kw) -> int:
     """
-    Test to verify if adequate logging is generated in the OSD logs upon initiating scrub & deep-scrubs
+    1.Test to verify if adequate logging is generated in the OSD logs upon initiating scrub & deep-scrubs
+    2. Verification of the logs if the mon_cluster_log_level values are warn and debug.More details are
+       updated at - https://bugzilla.redhat.com/show_bug.cgi?id=2053021
     Returns:
         1 -> Fail, 0 -> Pass
     """
```

Comment:
Make the bug verification optional via a flag passed from suite file

Reply:
Changed the logic and provide the flag.